website/util/__init__.py

```diff
@@ -69,7 +70,7 @@ def api_url_for(view_name, _absolute=False, _xml=False, *args, **kwargs):
 def api_v2_url(path_str,
                params=None,
                base_route=website_settings.API_DOMAIN,
-               base_prefix='',
+               base_prefix=api_settings.API_BASE,
```

Comment:
It might make sense to do:

```python
def api_v2_url(..., base_prefix=None, ...):
    base_prefix = base_prefix or api_settings.API_BASE
```

This would allow us to modify api_settings.API_BASE in tests, if we ever needed to.

```python
    @classmethod
    def setUpClass(cls):
        api_settings.API_BASE = 'api/test`
```

For similar usage, see the search functions (https://github.com/CenterForOpenScience/osf.io/blob/b9495e80a8f68549b3f2683d0120b948645f4a1a/website/search/search.py#L58-61) and tests (https://github.com/CenterForOpenScience/osf.io/blob/b9495e80a8f68549b3f2683d0120b948645f4a1a/tests/test_elastic.py#L35)

Not quite sure if it's necessary, but it's something to consider.